src/trix/models/html_sanitizer.js

```diff
@@ -1,29 +1,20 @@
+import * as config from "trix/config"
 import BasicObject from "trix/core/basic_object"
 
 import { nodeIsAttachmentElement, removeNode, tagName, walkTree } from "trix/core/helpers"
 
-const DEFAULT_ALLOWED_ATTRIBUTES = "style href src width height language class".split(" ")
-const DEFAULT_FORBIDDEN_PROTOCOLS = "javascript:".split(" ")
-const DEFAULT_FORBIDDEN_ELEMENTS = "script iframe form noscript".split(" ")
-
 export default class HTMLSanitizer extends BasicObject {
   static setHTML(element, html) {
     const sanitizedElement = new this(html).sanitize()
     const sanitizedHtml = sanitizedElement.getHTML ? sanitizedElement.getHTML() : sanitizedElement.outerHTML
     element.innerHTML = sanitizedHtml
   }
 
-  static sanitize(html, options) {
-    const sanitizer = new this(html, options)
-    sanitizer.sanitize()
-    return sanitizer
-  }
-
```

Comment:
I removed this function since it wasn't used, and to prevent any confusion with the non-static `sanitize` method